sdks/python/tox.ini

```diff
@@ -48,6 +48,11 @@ deps =
 whitelist_externals=find
 commands =
   python --version
+  # Clean up all previous cython generated files.
+  - find apache_beam -type f -name '*.c' -delete
```

Comment:
dash ignores the command exit code so you still see the error but it doesn't fail the test.